platform/security/encryption/platform-security-encryption-impl/src/main/java/ddf/security/encryption/impl/EncryptionServiceImpl.java

```diff
@@ -25,41 +27,59 @@
 public class EncryptionServiceImpl implements EncryptionService {
     private static final Logger LOGGER = LoggerFactory.getLogger(EncryptionServiceImpl.class);
 
-    private String key;
+    private static String passwordDirectory;
+
+    static Crypter crypter;
 
     public EncryptionServiceImpl() {
     }
 
-    /**
-     * Encrypts a plain text value using jasypt.
-     *
-     * @param plainTextValue
-     *            The value to encrypt.
-     */
-    public String encrypt(String plainTextValue) {
-        BasicTextEncryptor encryptor = new BasicTextEncryptor();
-        encryptor.setPassword(this.key);
-        return encryptor.encrypt(plainTextValue);
+    static {
+        passwordDirectory = System.getProperty("ddf.home").concat("/etc/certs");
+
+        if (!new File(System.getProperty("ddf.home").concat("/etc/certs/meta")).exists()) {
```

Comment:
Might want to do a passwordDirectory.concat("/meta"). I assume meta is what the KeyczarTool generates.